src/Mod/Draft/draftobjects/patharray.py

```diff
@@ -442,9 +470,12 @@ def placements_on_path(shapeRotation, pathwire, count, xlate, align,
         cdist += e.Length
         ends.append(cdist)
 
-    step = cdist / (count if DraftGeomUtils.isReallyClosed(pathwire) else count - 1)
+    start = getattr(startOffset, 'Value', startOffset)
+    end = getattr(endOffset, 'Value', endOffset)
```

Comment:
This is a bit cryptic. IMO an internal function like `placements_on_path` should not contain such code. It can be avoided by passing the `obj.StartOffset.Value` and `obj.EndOffset.Value` in the calling function.

Reply:
I guess the idea was to get a default value... But I think it's possible to do something like `obj.StartOffset.Value or startOffset`